packages/big-design/src/components/Button/styled.tsx

```diff
@@ -88,7 +88,11 @@ export const ContentWrapper = styled.span.attrs<Record<string, unknown>, { isLoa
   display: inline-grid;
   grid-auto-flow: column;
   grid-gap: ${({ theme }) => theme.spacing.xSmall};
-  visibility: ${({ isLoading }) => (isLoading ? 'hidden' : 'visible')};
+  ${({ isLoading }) =>
+    isLoading &&
+    css`
+      visibility: hidden;
+    `};
```

Comment:
Need to update this logic since this was causing that the style (visibility: hidden) was not applied for the buttons since they have by default `visibility: visible`.
s